src/cupbearer/tasks/tiny_natural_mechanisms.py

```diff
@@ -13,6 +14,54 @@
 from .task import Task
 
 
+def decode_and_encode(tokens, task_model, model):
+    decoded = task_model.tokenizer.decode(tokens)
+    reencoded = model.tokenizer.encode(decoded)
+    return reencoded
+
+def pad_tokens(tokens, pad_token_id, max_len=16):
+        tokens_len = len(tokens)
+        return [pad_token_id] * (max(max_len - tokens_len, 0)) + tokens[-min(tokens_len, max_len):]
+
+def get_effect_tokens(behavior_name, model):
+    from hex_nn.masking.behaviors import registry as behavior_registry
+    new_behavior = behavior_registry[behavior_name](model.tokenizer)
+    new_effect_tokens = list(new_behavior.effect_tokens)
+    return new_effect_tokens
+
+def convert_task_to_model(behavior_name, model_name, task_data, task_model, model):
```

Comment:
Similarly, `old_model` and `new_model` or something similar might be clearer (and useful here since this function may be  a bit more user-facing)